app/models/manageiq/providers/embedded_ansible/crud_common.rb

```diff
@@ -50,7 +50,6 @@ def encrypt_queue_params(params)
     end
 
     def create_in_provider_queue(manager_id, params, auth_user = nil)
-      manager = parent.find(manager_id)
```

Comment:
Interesting - @agrare ?

Reply:
Yeah @djberg96 posted about this on gitter this morning [[ref]](https://gitter.im/ManageIQ/manageiq-providers-embedded_ansible?at=5fcf87418d1477540fad13f6), https://github.com/ManageIQ/manageiq/commit/34eac12841d18b3d3839adf6edff60c7ec3c110e#diff-5d49121ce998251e7bee6e65be3367227fc6caddb651caa82f198a188458de73L56 removed the use of `manager.my_zone` so this lookup isn't necessary anymore